.github/workflows/e2e-tests-wp-nightly.yml

```diff
@@ -1,11 +1,48 @@
 name: E2E Tests
 
+defaults:
+  timeout-minutes: 20
+
 on:
   push:
     branches:
       - develop
       - main
+    paths-ignore:
```

Comment:
I think we should probably always include a `paths` list of patterns rather than ignore only.

Maybe this would be:
* assets/**
* includes/**
* /google-site-kit.php
* lock files (any dependency changes)
* config files

While continuing to ignore tests

That way, if we add some new file outside of the ignore patterns it wouldn't trigger E2E.